src/cli/dsc/schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Oh My Posh DSC State",
+  "description": "State configuration for Oh My Posh DSC (Desired State Configuration)",
+  "properties": {
+    "configurations": {
+      "type": "array",
+      "title": "Configuration List",
+      "description": "List of Oh My Posh configurations",
+      "items": {
+        "type": "object",
+        "properties": {
+          "path": {
+            "type": "string",
+            "title": "Configuration Path",
+            "description": "File path to the Oh My Posh configuration file"
+          },
+          "format": {
+            "type": "string",
+            "title": "Configuration Format",
+            "description": "Format of the configuration file",
+            "enum": [
+              "json",
+              "yaml",
+              "toml"
+            ]
+          },
+          "data": {
+            "title": "Configuration Data",
+            "description": "The actual Oh My Posh configuration object",
+            "$ref": "https://ohmyposh.dev/schema.json"
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "shells": {
+      "type": "array",
+      "title": "Shell List",
+      "description": "List of installed shells using Oh My Posh",
```

Comment:
Is this intended to be configurable from the resource, or just informational? If the latter, you'll want to define the `readOnly` keyword as `true` to indicate the non-configurable nature of this property.

Reply:
This can be configured. That logic is implemented.